test/unit/database/test_build_store.py

```diff
@@ -0,0 +1,142 @@
+from collections import MutableMapping
+from concurrent.futures import ThreadPoolExecutor
+import os
+from os.path import isfile
+import re
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sys import maxsize
+
+from genty import genty, genty_dataset
+from unittest.mock import MagicMock, Mock
+
+from app.database.build_store import BuildStore
+from test.framework.base_unit_test_case import BaseUnitTestCase
+from app.master.atomizer import Atomizer
+from app.master.build import Build, BuildStatus
+from app.master.build_request import BuildRequest
+from app.master.cluster_master import ClusterMaster
+from app.master.job_config import JobConfig
+from app.project_type.project_type import ProjectType
+from app.util.conf.configuration import Configuration
+
+
+TEST_DB_NAME = 'test.db'
+TEST_DB_URL = 'sqlite:///test.db'
+
+
+@genty
+class TestBuildStore(BaseUnitTestCase):
+    def setUp(self):
+        super().setUp()
+        self.patch('app.util.fs.create_dir')
+        self.patch('app.util.fs.async_delete')
+        self.patch('os.makedirs')
```

Comment:
You probably don't want to patch `os` methods directly. Instead patch through the module you are trying to mock that imports `os`.